commit 4071a44f8ce6cfc7eabe5f68db9958ed9a714c1d (refs/changes/77/277/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-12T00:06:48+00:00 (3 years, 2 months ago)
    
    joyent/node-verror#29 No documentation on how to Install this package
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index f6dc25c..4839f1a 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,4 +2,5 @@
 
 * #10 better support for extra properties on Errors
 * #11 make it easy to find causes of a particular kind
+* #29 No documentation on how to Install this package
 * #36 elide development-only files from npm package
diff --git a/README.md b/README.md
index 902bdf3..5d34a53 100644
--- a/README.md
+++ b/README.md
@@ -30,6 +30,10 @@ The classes here are:
 
 # Quick start
 
+First, install the package:
+
+    npm install verror
+
 If nothing else, you can use VError as a drop-in replacement for the built-in
 JavaScript Error class, with the addition of printf-style messages:
 
